### PR TITLE
Fix build context

### LINKED ISSUE #1410: Building docker-compose file with absolute path fail
# What happened:
There is unable to build when I build image with absolute path of compose file.
```
$ kompose convert --build=local -f /Users/clixin/apps/source/kompose/script/test/fixtures/buildconfig/docker-compose.yml
INFO Build key detected. Attempting to build image 'docker.io/cdrage/foobar' 
FATA Unable to build Docker image for service foo: Unable to build image. For more output, use -v or --verbose when converting.: API error (500): Cannot locate specified Dockerfile: Dockerfile
```
# What you expected to happen:
This works when I use relative path of compose file.
```
$ kompose -f ./script/test/fixtures/buildconfig/docker-compose.yml convert --build=local
INFO Image 'docker.io/cdrage/foobar' from directory 'build' built successfully
```
# Anything else we need to know?:
The docker compose file I use.
```
version: "2"

services:
    foo:
        build: "./build"
        image: docker.io/cdrage/foobar
```

When add `-v` to see the debug output from building with absolute path.
```
$ kompose -f /Users/clixin/apps/source/kompose/script/test/fixtures/buildconfig/docker-compose.yml convert --build=local -v
DEBU Checking validation of provider: kubernetes  
DEBU Checking validation of controller:           
DEBU Docker Compose version: 2                    
DEBU Opening compose files: /Users/clixin/apps/source/kompose/script/test/fixtures/buildconfig/docker-compose.yml 
DEBU [0/1] [foo]: Adding                          
DEBU [0/1] [default]: EventType: 32               
DEBU Default network found                        
INFO Build key detected. Attempting to build image 'docker.io/cdrage/foobar' 
DEBU Build image working dir is: /Users/clixin/apps/source/kompose 
DEBU Build image context is: /Users/clixin/apps/source/kompose/build 
INFO Building image 'docker.io/cdrage/foobar' from directory 'build' 
DEBU Created temporary file /var/folders/x1/btblng4s7qv3x2dhf2yhbdgh0000gn/T/kompose-image-build-072047000 for Docker image tarballing 
DEBU Image docker.io/cdrage/foobar build output:  
FATA Unable to build Docker image for service foo: Unable to build image. For more output, use -v or --verbose when converting.: API error (500): Cannot locate specified Dockerfile: Dockerfile
```

The build image working context is not resolved correctly.
```
DEBU Build image context is: /Users/clixin/apps/source/kompose/build 
```

I will make PR to resolve it.

### PR DESCRIPTION
**Pull request by lexcao**
Fix #1410

### CODE REVIEW

File: script/test/cmd/tests_new.sh [hunk 1]
```diff
@@ -82,3 +82,16 @@ convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/script/test/fixture
 #TEST the pvc-request-size command parameter
 convert::check_artifacts_generated "kompose -f $KOMPOSE_ROOT/script/test/fixtures/pvc-request-size/docker-compose.yml convert -o $TEMP_DIR/output_dir2/output-k8s.json -j --pvc-request-size=300Mi" "$TEMP_DIR/output_dir2/output-k8s.json"
 convert::check_artifacts_generated "kompose --provider=openshift -f $KOMPOSE_ROOT/script/test/fixtures/pvc-request-size/docker-compose.yml convert -o $TEMP_DIR/output_dir2/output-os.json -j --pvc-request-size=300Mi" "$TEMP_DIR/output_dir2/output-os.json"
+
+######
+# Test the path of build image
+# Test build v2 absolute compose file
+convert::check_artifacts_generated "kompose --build local -f $KOMPOSE_ROOT/script/test_in_openshift/compose-files/buildconfig/docker-compose.yml convert -o $TEMP_DIR/output_file" "$TEMP_DIR/output_file"
```
Comment (hangyan) [inline script/test/cmd/tests_new.sh:89]: Most of the testdir has been deprecated, try using `script/fixtures/buildconfig/` as your fixtures.

Comment (lexcao) [reply script/test/cmd/tests_new.sh]: Ok, I have fixed it.

**Review by hangyan — APPROVED** [review]

**Pull request merged**